.pre-commit-config.yaml

```diff
@@ -1,11 +1,12 @@
 repos:
 -   repo: https://github.com/psf/black
-    rev: stable
+    rev: 20.8b1
```

Comment:
I think letting black float is fine unless it's been a big source of pain in the past?

Reply:
Historically it's been a slight pain point, with a mismatch between local `black` and `black` in CI. Though it sounds like that may have been related to your changes in https://github.com/dask/dask/pull/6641

Reply:
right but there would not longer be a mismatch right?

Reply:
:facepalm: I see now that you changed it. Thanks!